tools/enrollment_preprocessor.py

```diff
@@ -16,63 +20,89 @@ class User:
     first_name: str
     email: str
 
+    def full_name(self) -> str:
+        prefix = ""
+        if self.title:
+            prefix = f"{self.title} "
+        return f"{prefix}{self.first_name} {self.last_name}, {self.email}"
 
-@dataclass
-class UserCells:
+
+class UserCells(NamedTuple):
     title: Cell | None
     last_name: Cell
     first_name: Cell
     email: Cell
 
-    def value(self) -> User:
-        return User(
-            str(self.title.value) if self.title and self.title.value else "",
-            str(self.last_name.value),
-            str(self.first_name.value),
-            str(self.email.value),
-        )
-
-
-def user_decision(field: str, existing: str, imported: str) -> str:
-    if existing == imported:
-        return existing
-    decision: str = ""
-    while decision not in ("y", "n"):
-        decision = input(f"Do you want to keep the existing user {field}? (y/n) ")
-    if decision == "n":
-        return imported
-    return existing
-
-
-def fix_user(users: dict[str, User], imported_cells: UserCells) -> None:
-    imported = imported_cells.value()
-    existing = users.setdefault(imported.email, imported)
-    if imported == existing:
-        return
-    print("There is a conflict in the user data.")
-    print(f"existing: {existing}.")
-    print(f"imported: {imported}.")
-    # None is passed exclusively for participants since they have no title column
-    if imported_cells.title is not None:
-        imported_cells.title.value = user_decision("title", existing.title, imported.title)
-    imported_cells.last_name.value = user_decision("last name", existing.last_name, imported.last_name)
-    imported_cells.first_name.value = user_decision("first name", existing.first_name, imported.first_name)
-    imported_cells.email.value = user_decision("email", existing.email, imported.email)
-    print()
-
-
-def run_preprocessor(enrollment_data: str | BytesIO, user_data: TextIO) -> BytesIO:
-    workbook = load_workbook(enrollment_data)
+    def clean(self) -> Iterator[str]:
+        for field in iter(self):
+            if not field or not field.value:
+                yield ""
+                continue
+            field.value = str(field.value or "").strip()
+            yield field.value
+
+
+def make_bold(text: str) -> str:
+    return f"\x1b[1m{text}\x1b[0m"
```

Comment:
I think I'm more used to the octal representation of the escape code (`\033[1m`), but no strong opinion either